riscv/src/compiler.rs

```diff
@@ -176,8 +183,9 @@ fn store_data_objects<'a>(
                 }
                 pos += item.size() as u32;
             }
-            let first_line = object_code.first_mut().unwrap();
```

Comment:
Why did this not panic before?

Reply:
`object_code` was not empty before, now it can because `DataValue::Zero` does not add anything to `object_code`.